mslib/msui/mscolab_version_history.py

```diff
@@ -134,10 +133,9 @@ def load_all_changes(self):
                 "token": self.token,
                 "op_id": self.op_id
             }
-            named_version_only = None
```

Comment:
this was one of these abuse of None for boolean. The minimum change would be here to set it to False.

The coresponding server code should also become fixed, e.g. file_manager.get_all_changes

Reply:
Done